src/devices/CharacterLcd/IGpioController.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+
+namespace Iot.Device.CharacterLcd
+{
+    // Can we move this upstream and actually derive from this? Should I2C/SPI derive from GpioDriver instead?
+    // Having an interface would allow for easier unit testing...
+    public interface IGpioController : IDisposable
```

Comment:
Should we force this interface to be disposable? I'm not 100% sure that is required and applicable in all cases.

Reply:
You don't have to do anything on it if your controller doesn't have to be cleaned up. Some do.